modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3107,14 +3107,8 @@ def dropna(self, **kwargs):
             lib.no_default,
             None,
         )
-        # FIXME: this is a naive workaround for this problem: https://github.com/modin-project/modin/issues/5394
-        # if there are too many partitions then all non-full-axis implementations start acting very badly.
-        # The here threshold is pretty random though it works fine on simple scenarios
-        processable_amount_of_partitions = (
-            self._modin_frame.num_parts < CpuCount.get() * 32
-        )
```

Comment:
Why is this removed?

Reply:
As you can see from this comment, this condition is a naive workaround for a problem that is solved by dynamic partitioning in this PR.